pkg/api/v1/attribute_list_params.go

```diff
@@ -43,15 +56,24 @@ func encodeAttributesListParams(alp []AttributeListParams, key string, q url.Val
 			}
 		}
 
+		if ap.AttributeOperator != "" {
+			value += "~" + string(ap.AttributeOperator)
+		}
+
 		q.Add(key, value)
 	}
 }
 
 func parseQueryAttributesListParams(c *gin.Context, key string) []AttributeListParams {
 	alp := []AttributeListParams{}
 
-	for _, p := range c.QueryArray(key) {
-		// format is "ns~keys.dot.seperated~operation~value"
+	attrQueryParams := c.QueryArray(key)
+
+	for _, p := range attrQueryParams {
+		// format accepted
+		// "ns~keys.dot.seperated~operation~value"
+		// With attr OR operator: "ns~keys.dot.seperated~operation~value~or"
+		// With attr AND operator: "ns~keys.dot.seperated~operation~value~and"
```

Comment:
It would be nice to have this comment somewhere in the docs, maybe as part of the documentation for `AttributeListParams`. What do you think? As a user I usually find it hard to figure out the query string required to filter results in ServerService.

Reply:
I agree, I always end up looking at the tests to figure out about the query string format.

Although, `AttributeListParams` is encoded into the string format by the client, it won't be as helpful to have those comments on that struct imo, We probably need a cheatsheet or some other doc generation mechanism that can be used for reference.

Reply:
It's the only thing that gets exposed to the user but I agree is not the best place. I suppose leave as is since we have to always dive into the code to use it